benchmarks/benchmarks/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
In the terminal PR, we'll probably ask you to move these benchmarks to https://github.com/pytorch/benchmark since it makes it easier to run benchmarks across versions if they live out of line.

Reply:
Ah, that's where they live - I was wondering why there were so few in this repo. Sounds good to move these at the end.